depthai_nodes/ml/parsers/parser.py

```diff
@@ -1,10 +1,24 @@
-from typing import Dict, List, Union
+from typing import Dict, List, Union, Optional
 
```

Comment:
Should we move this parser one folder higher so that the import is nice and more logcal?

Reply:
I think it's fine to leave this with all other parsers. In most cases users shouldn't import the this base parser class anyway. And we can also add `__all__` in the `__init__` to control "niceness" of the imports - can be addressed down the line.